### PR TITLE
docs: updating documentation to link quickstart and main docs

### PR DESCRIPTION
**Pull request by bernadinm**
Updating the documentation to sync the flow of using the main.tf.

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -18,10 +18,20 @@ mkdir dcos-installer
 cd dcos-installer
 ```
 
+## Initialize Terraform
+
 Run this command below to have Terraform initialized from this repository. There is **no git clone of this repo required** as Terraform performs this for you.
```
Comment (anithamuthu79) [inline docs/INSTALL.md]: Use the following command to initialize Terraform from this repository. There is no git clone of this repo required as Terraform performs this for you.

File: docs/INSTALL.md [hunk 2]
```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
...
+#### Supported DC/OS Versions
+
+Here is the list of DC/OS versions supported on dcos-terraform natively:
 
-The default variables are tracked in the [variables.tf](../variables.tf) file. Since this file can be overwritten during updates when you may run `terraform get --update` when you want to fetch new releases of DC/OS to upgrade too, its best to use the cluster_profile.tfvars and set your custom terraform and DC/OS flags there. This way you can keep track of a single file that you can use manage the lifecycle of your cluster.
+- [OSS Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/open/dcos-versions)
+- [Enterprise Versions](https://github.com/dcos-terraform/terraform-template-dcos-core/tree/master/ee/dcos-versions)
 
-To apply the configuration file, you can use this command below.
+**Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
```
Comment (anithamuthu79) [inline docs/INSTALL.md]: <p class="message--note"><strong>NOTE: </strong>Master DC/OS version is not meant for production use. It is only for CI/CD testing.</p>

Comment (anithamuthu79) [reply docs/INSTALL.md]: Remove content from line #61 and copy-paste the above line.

File: docs/INSTALL.md [hunk 3]
```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
...
-# we can set the azure location
-echo 'region = "us-west1"' >> cluster_profile.tfvars
-# set the google project id
-echo 'project_id = "massive-bliss-781"' >> cluster_profile.tfvars
+terraform apply
+```
+
+## Advanced YAML Configuration
+
+We have designed this project to be flexible. Here are the example working variables that allows very deep customization by using a single `main.tf` file.
```
Comment (anithamuthu79) [inline docs/INSTALL.md]: Rewrite as:
The configuration templates using Terraform are designed to be flexible. Here is an example of working variables that allows deep customization by using a single main.tf file.

File: docs/INSTALL.md [hunk 4]
```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
...
+
+For advanced users with stringent requirements, here are DC/OS flag examples you can simply paste in `main.tf` file.
```
Comment (anithamuthu79) [inline docs/INSTALL.md]: Rewrite as:
For advanced users with stringent requirements, paste the DC/OS flag examples in main.tf file.

File: docs/INSTALL.md [hunk 5]
```diff
@@ -37,29 +47,87 @@ $ gcloud auth login
...
+          ignoreusedcount: true
+  EOF
+  dcos_cluster_docker_credentials = <<EOF
+  # YAML
+    auths:
+      'https://index.docker.io/v1/':
+        auth: Ze9ja2VyY3licmljSmVFOEJrcTY2eTV1WHhnSkVuVndjVEE=
+  EOF
 ```
+**Note**: The YAML comment is required for the DC/OS specific YAML settings.
```
Comment (anithamuthu79) [inline docs/INSTALL.md]: <p class="message--note"><strong>NOTE: </strong>The YAML comment is required for the DC/OS specific YAML settings.</p>

Comment (anithamuthu79) [reply docs/INSTALL.md]: Remove the content in line #165 and copy-paste the above line.

File: docs/quickstart/README.md [hunk 6]
```diff
@@ -337,6 +337,8 @@ Once the apply completes, you can verify that the cluster was upgraded via the D
 <img src="../images/upgrade/cluster-details-open.png" />
 </p>
 
+# Maintenance
+ For instructions on how to maintain your cluster, follow the [maintenance](https://github.com/dcos-terraform/terraform-gcp-dcos/blob/master/docs/MAINTAIN.md) documentation.
 
 # Deleting Your Cluster
 If you ever decide you would like to destroy your cluster, simply run the following command and wait for it to complete:
```
Comment (anithamuthu79) [inline docs/quickstart/README.md]: Rewrite as:
If you want to destroy your cluster, then use the following command and wait for it to complete.

File: docs/DESTROY.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 You can shutdown/destroy all resources from your environment by running this command below:
```
[the remark this answers is not in the document]
Comment (bernadinm) [reply docs/DESTROY.md]: Done

File: docs/INSTALL.md (hunk 1 above)
Comment (bernadinm) [reply docs/INSTALL.md]: Done

**Comment by bernadinm** [comment]
@anithamuthu79, I've made the changes you've requested here. Please let me know if it looks good to you again. tymv!

**Comment by anithamuthu79** [comment]
@bernadinm as per our discussion, pls update the limitations section on or before Monday(10/22).

**Comment by bernadinm** [comment]
Hi @anithamuthu79, docs are updated with just chandlers changes that we've discussed. The Features and limitations will happen in another pr. You can feel free to accept the changes and we can merge it.

**Pull request merged**